Fuzzy or implicit plurality fix (#1093)
Bittu5134/ORV-Reader: chapters/orv/chap_00103.txt

<!>[The constellation 'Maritime War God' is touched by your companionship.]
<!>[The constellation 'Bald General of Justice' is blushing at your companionship.]
<!>[500 coins have been sponsored.]

> Yes, now I understood. Yoo Joonghyuk, he was looking for donations. It was a little strange to see that Uriel wasn't involved, but he should be aiming for a sponsorship by one of the constellations.

Reviewer: ```suggestion
Yes, now I understood. Yoo Joonghyuk, he was looking for donations. It was a little strange to see that Uriel wasn't involved, but he must be aiming for sponsorships by the constellations.
```
Felt like YJH would be aiming for multiple sponsorships instead of just one

This could be the only outcome. The 41st regression Shin Yoosung…
She never once heard Yoo Joonghyuk call someone his companion. Shin Yoosung's fist hit Yoo Joonghyuk's blade.
Despite the clash between the body and the weapon, what received damage was Yoo Joonghyuk's sword, the SS-grade Splitting the Sky Sword. Putting aside the special options, its durability and strength were one of the best to exist, and yet... this sword was damaged by Shin Yoosung's fist.
In the end, the Splitting the Sky Sword failed to overcome the shock, was bent and helplessly deflected.
<&>「 "How dare you say this in front of me?" 」
Shin Yoosung shouted and didn't use any special skills. She just threw a punch that contained ether condensed to the limit.
Yoo Joonghyuk coughed up blood and flew back at this blow. Her attacks, her speed and her variations. Shin Yoosung was superior to Yoo Joonghyuk in all aspects.
The Red Phoenix Shunpo and Splitting the Sky Sword that Yoo Joonghyuk was proud of were dealt with like a flickering flame in front of Shin Yoosung.
There was the terrible sound of the body breaking and Yoo Joonghyuk was pushed back. It wasn't a matter of talent but a matter of time.

> Despite being weakened by the descent, the 41st Shin Yoosung Beast Lord was near the peak of her power. On the other hand, the current Yoo Joonghyuk was only in the first stages of his growth.

Reviewer: ```suggestion
Despite being weakened by the descent, the 41st Shin Yoosung, the Beast Lord was near the peak of her power. On the other hand, the current Yoo Joonghyuk was only in the early stages of his growth.
```

Added emphasis on beast lord, and replaced first to early dive YJH has finished 3 other regressions before the current one

Author: Looks good, however wouldn't the emphasis on "Beast Lord" break the coherence of other "41st Shin Yoosung Beast Lord" occurences?

Reviewer: Maybe, but even the original text tried to put emphasis on beast lord

<&>「 "You can't be like this…" 」
An ominous aura filled her surroundings. It was a process where emptiness turned to betrayal and betrayal turned to anger.
<&>「 "You are changing. I can't tolerate it." 」
Ether filled Shin Yoosung's fists. I wanted to watch a bit more, but a sense of crisis came over me.
<!>[Excessive immersion leads to a profound increase in the proficiency of Omniscient Reader's Viewpoint.]
If this went wrong, then Yoo Joonghyuk would really die. I couldn't let him be killed here. I quickly switched the mode of Omniscient Reader's Viewpoint.
From 3rd person to 1st person protagonist's point of view.
<!>[Changing your viewpoint to 1st person.]
…
<!>[Changing to the 1st person protagonist's point of view has failed.]
What? Why?
<!>[You have failed to meet the conditions to change the viewpoint.]
It felt like I had been slammed in the back of the head. There were two conditions to use 1st person protagonist's point of view.
One, I must be dead and in an out of body state. Two, the target and I should be thinking about each other.

> The first condition was met but the problem was with the second condition. That jerk isn't thinking about me right now?

Reviewer: ```suggestion
The first condition was met but the problem was with the second condition. That jerk wasn't thinking about me right now?
```
Wasn't feels more accurate here

Author: Hmmm, I see. He is talking in the past tense now that I re-think it.

Then what's with all the running amok because of revenge? I watched Yoo Joonghyuk frantically attacking and it dawned on me.
<&>「 "Die. I will kill you." 」

> …That bastard, was he thinking about anything? Shin Yoosung glared at Yoo Joonghyuk and opened her mouth.

Reviewer: ```suggestion
…That bastard, was he thinking about anything? Shin Yoosung watched Yoo Joonghyuk and opened her mouth.
```
Watched should be used here because shin yoosung is pitying, tho I am open to suggestions

Author: Is she though? Maybe, I'll look at it again. The devil smile soon after does kind of break away from the pitying branch of emotions though.

Author: Hmmm, that's a good point. Agreed.

The 5th grade sea species, King Masswood. The 5th grade behemoth, Heavy Metal Bean.
They were monsters who could cause small disasters in this world. She was about to speak when a sharp strike came from the side.
"Where are you going?"

> It was a flashy sword. The person wielding it wore a black hoodie over a slim skirt. Shin Yoosung immediately recognized who she was. The Demon Slayer was burning in Lee Jihye's eyes.

Reviewer: ```suggestion
It was a flashy sword. The person wielding it wore a black hoodie over a slim skirt. Shin Yoosung immediately recognized who she was. Demon Slaying was burning in Lee Jihye's eyes.
```
It's talking about the [Demon Slaying] skill

There should be brackets here but the initial chapters ignored those brackets and it would be weird to just add them at one place

Author: Oooohh, that's why I was confused. Thanks, might be worth opening an issue about this.

Reviewer: From initial chapters i mean the first 300 chapters ignored them, it's too much work to edit as of them

"You dared to make Master look like that?"
The majesty of a great constellation emerged from Lee Jihye's body, the might of someone overpowering on the sea. Shin Yoosung noticed the stigma she intended to use. This place was a river.
"…Your Majesty."
The moment she started reciting a passage, transparent ships emerged from every corner of the Han River.
"I still have 12 ships…!"

> 12 ships appeared on the Han River, scattering the waves and current. It was a sight that made people speechless. The stigma of the Duke of Loyalty and Warfare, Ghost Fleet. Despite the oppressive aura of the ships, Shin Yoosung only smiled like she had missed it.

Reviewer: ```suggestion
12 ships appeared on the Han River, scattering the waves and current. It was a sight that made people speechless. The stigma of the Duke of Loyalty and Warfare, Ghost Fleet. Despite the oppressive aura of the ships, Shin Yoosung only smiled as if she had missed it.
```

Author: Flows better, agreed.
